src/platform_impl/macos/window.rs

```diff
@@ -424,7 +433,7 @@ impl UnownedWindow {
             .into())
```

Comment:
I believe this needs to be converted from logical coordinates to physical coordinates.